Final_project_2/frontend/index.html

```diff
@@ -347,49 +346,43 @@ <h6>We love talking and hearing from you</h6>
                     </div>
                 </div>
             </div>
-        <div>
-            <address class="information">
-            <span>
+        <div class="address-block">
```

Comment:
Had to rework the structure of the divs here a bit so that the centering would work as expected.